spec/rspec/mocks/and_call_original_spec.rb

```diff
@@ -143,6 +143,19 @@ def inst.foo; :bar; end
       expect(subclass.new_instance).to be_a(subclass)
     end
 
+    context 'when a class method is stubbed in the superclass' do
+      it 'still works for class methods defined on a superclass' do
```

Comment:
We have 2 different specs here:

- Class method defined in superclass, inherited by subclass, stubbed in subclass
- Class method defined in superclass, inherited by subclass, *stubbed in superclass*, stubbed separately in subclass

Any suggestions on a better docstring to convey the difference between these 2 examples?